ZichanYang324/BF-Employee-HR
# BGP-49 [Frontend and Backend] HR-Onboarding Application Review

# Description
Implements BGP-49, onboarding application review of hr side

## Related PRS (if any):
Based on #36 , please review #36 first.
…

## Main changes explained:
- View all submitted onboarding applications
- Check each application's details
- Approve or reject applications, and provide feedback
…

## How to test:
1. check into current branch
2. Click 'Hiring Management' on nav bar  and 'application review' on side bar
3. Test functionalities

## Screenshots or videos of changes or expecting result:

![image](https://github.com/ZichanYang324/BF-Employee-HR/assets/85213835/25416c72-83b1-4335-aca9-1350e1358362)

![image](https://github.com/ZichanYang324/BF-Employee-HR/assets/85213835/837e088f-c8b3-4e64-b504-955fdb6c2384)


## Note:
Review and merge #36 before this PR
